frontend/static/md_pages/tmc-cli_installation_Windows_en.mdx

```diff
@@ -0,0 +1,107 @@
+import {ContentBox, SectionBox} from "/pages/[lng]/installation/netbeans"
+
+<ContentBox>
+
+## Installing to Linux
+---
+### Contents
+1. [Install using installer](#using-installer)
+2. [Install manually](#manual-installation)
+3. [Getting started](#getting-started)
+---
+
+</ContentBox>
+
+<ContentBox>
+
+Follow the instructions below to install the Test My Code client
+
+<SectionBox>
+
+### Install using the installer<a name="using-installer"></a>
+
+Download the installer 'tmc-cli-rust.msi' at ... TODO
```

Comment:
Is this supposed to go out as TODO?

Reply:
It is not finished yet, I was asked by Henrik to make a PR of the current instructions